package.json

```diff
@@ -23,7 +23,7 @@
     "@mate-academy/bemlint": "^0.1.1",
     "@mate-academy/eslint-config": "*",
     "@mate-academy/linthtml-config": "0.0.2",
-    "@mate-academy/scripts": "^0.9.14",
+    "@mate-academy/scripts": "^1.2.8",
```

Comment:
The version of @mate-academy/scripts has been updated from 0.9.14 to 1.2.8. Make sure the new version is compatible with your project and doesn't cause any issues. 🚀